element_array_ephys/export/nwb/tests.py

```diff
@@ -0,0 +1,37 @@
+from element_array_ephys.export.nwb.nwb import ecephys_session_to_nwb, write_nwb
+
+from pynwb.ecephys import ElectricalSeries
+
+
+def test_convert_to_nwb():
+
+    nwbfile = ecephys_session_to_nwb(
+        dict(subject="subject5", session_datetime="2020-05-12 04:13:07")
```

Comment:
Does this test covers the `add_ephys_lfp_from_dj_to_nwb` function or the upstream lab/project/protocol pieces. I think it would be helpful for future integrations to make sure we haven't broken those links (e.g., changes to our table architecture that needed corresponding changes to the export functions).
```suggestion
        dict(subject="subject5", session_datetime="2020-05-12 04:13:07",lfp="dj")
```

Reply:
I agree it would be useful to test that case too. This isn't the best way to do it though. I'll push an addition for that in the next few days.

Reply:
@CBroz1 I have added this test